patches/gcc-10.1.0/0008-s390x-muslldso.diff

```diff
@@ -0,0 +1,14 @@
+diff --git a/gcc/config/s390/linux.h b/gcc/config/s390/linux.h
+index 6919b468f06..95881250e23 100644
+--- a/gcc/config/s390/linux.h
++++ b/gcc/config/s390/linux.h
+@@ -75,6 +75,9 @@ along with GCC; see the file COPYING3.  If not see
+ #define GLIBC_DYNAMIC_LINKER32 "/lib/ld.so.1"
+ #define GLIBC_DYNAMIC_LINKER64 "/lib/ld64.so.1"
+ 
++#define MUSL_DYNAMIC_LINKER32 "/lib/ld-musl-s390.so.1"
++#define MUSL_DYNAMIC_LINKER64 "/lib/ld-musl-s390x.so.1"
```

Comment:
This has been [added upstream](https://gcc.gnu.org/git/?p=gcc.git;a=commit;h=3d6d8099b639b13760cab58720136a438ff68f0a), along with [another fix](https://gcc.gnu.org/git/?p=gcc.git;a=commit;h=754f54aa01975a8e2fa4e38850f90dcda7dd02ec).

Reply:
You are right.